app/views/content/train-to-be-a-teacher/how-to-choose-your-teacher-training-course.md

```diff
@@ -0,0 +1,137 @@
+
+[Find out how to apply for your visa to train to teach in England](/non-uk-teachers/visas-for-non-uk-trainees).
+
+## The qualifications awarded by the course
+
+Most schools in England need you to have [qualified teacher status (QTS)](/what-is-qts) to teach. 
+
+Most teacher training courses will award QTS, but some will just award a [postgraduate certificate in education (PGCE)](/what-is-a-pgce), and some will award both. 
+
+Without QTS, you will not eligible for a scholarship or bursary to help you train. You also may not be fully qualified to teach in your chosen school and will not receive the same pay and support when you start teaching.
+
+## If the course is fee-funded or salaried
+
+### Fee-funded teacher training
+
+Most teacher training courses are fee-funded – this means you have to pay tuition fees and will not earn a salary while you train. 
+
+There are ways to fund your training, for example, through tuition fee and maintenance loans and scholarships and bursaries. 
+
+You can [find out about the different ways to fund your training](/funding-and-support).
+
+<div class="inset">
+
+<p> If you’re a non-UK citizen without indefinite leave to remain in the UK, you will not usually be eligible for a bursary or scholarship, unless you train to teach languages or physics.</p>
+
+</div>
+
+### Salaried teacher training
+
+Some courses are salaried, which means you’ll receive an unqualified teacher’s salary while you train. 
+
+Places on salaried courses are limited and very competitive. You can [find out what you’ll need to get on a salaried teacher training course](/funding-and-support/salaried-teacher-training). 
+
+<div class="inset">
+
+<p>If you’re a non-UK citizen, the number of places on salaried courses is very limited.</p>
+
+</div>
+
+## The type of course provider 
+
+Your teacher training course might be provided by: 
+
+* a university (sometimes referred to as university-led training) 
+* a school or group of schools (sometimes referred to as school-led training, or an apprenticeship) 
+* [Teach First](https://www.teachfirst.org.uk/) (a charitable organisation)  
+
+Once you’ve found a course that you’re interested in, you can talk to the training provider before you apply. For example, you might want to find out: 
+
+* where your placements and theoretical learning might happen
+* how many schools you’ll be doing placements in and how long each placement will last 
+* the start and finish dates of the course 
+
+It’s a good idea to apply to a range of providers to increase your chances of being successful on a course. 
+
+For example, if you’re applying for 4 courses, you might want to apply for 2 courses provided by universities, and 2 provided by schools.
+
+<div class="inset">
+<p>If you’re a non-UK citizen, you might also want to talk to the training provider to confirm:</p>
+  <ul>
+     <li>if they offer visa sponsorship</li>
+     <li>the timing for non-UK applications</li>
```

Comment:
I understand why this is important but think we should also check with Apply about how to word this?